src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -377,7 +377,15 @@ public override DownloadClientInfo GetStatus()
             {
                 if (Proxy.GetLabels(Settings).TryGetValue(Settings.TvCategory, out var label) && label.SavePath.IsNotNullOrWhiteSpace())
                 {
-                    var labelDir = new OsPath(label.SavePath);
+                    var savePath = label.SavePath;
+
+                    if (savePath.StartsWith("//"))
+                    {
+                        _logger.Trace("Replacing slashes in probable Windows UNC path '{0}'. If this is not meant to be a Windows UNC path fix your qBittorrent configuration", savePath);
```

Comment:
```suggestion
                        _logger.Trace("Replacing double back slashes in probable Windows UNC path '{0}'. If this is not meant to be a Windows UNC path fix your qBittorrent configuration and don't use // for paths on non-Windows systems in Qbit", savePath);
```

wordy but can likely tweak

Reply:
Wrong slash, but definitely too wordy regardless.

Reply:
I updated it to point them at the Save Path for the category, if they can't get there from them then there isn't much we can do to spell it out much more clearly.